tests/install/test.py

```diff
@@ -427,6 +431,7 @@ def test_upgrade(self, firmware, create_vms, iso_remaster, orig_version, iso_ver
         lambda version: {
             "821.1": "xcpng-8.2.1-2023",
             "83rc1": "xcpng-8.3-rc1",
+            "83rc1-net": "xcpng-8.3-rc1-net",
```

Comment:
would be worth adding to `test_restore` and their respective firstboot tests at the same time